package.json

```diff
@@ -58,18 +58,18 @@
     "@ngrx/router-store": "~15.0.0",
     "@ngrx/store": "~15.0.0",
     "@nguniversal/builders": "~15.1.0",
-    "@nrwl/cypress": "15.6.0-beta.1",
-    "@nrwl/devkit": "15.6.0-beta.1",
-    "@nrwl/eslint-plugin-nx": "15.6.0-beta.1",
-    "@nrwl/jest": "15.6.0-beta.1",
-    "@nrwl/js": "15.6.0-beta.1",
-    "@nrwl/linter": "15.6.0-beta.1",
-    "@nrwl/next": "15.6.0-beta.1",
+    "@nrwl/cypress": "15.7.0-beta.3",
```

Comment:
I think you didn't mean to commit `package.json` and `yarn.lock`

Reply:
Well, yes. If @FrozenPandaz is fine with upgrading to the latest beta version, because otherwise right now the docs don't serve, at least locally. not sure whether that would potentially even propagate into deploying & breaking them in prod as well.

I can separate it out into a different PR though in case

Reply:
Seems like there's another PR in progress already, so I let that one merge first: https://github.com/nrwl/nx/pull/14834